### ISSUE TITLE
Allow install from seperate repo like bower

### ISSUE THREAD

**Issue by LordZardeck** [DefinitelyTyped/tsd#87]
In bower if there is a repo you want to include (like your own) that isn't listed in bower, you can install a repo by it's git url. It would be nice to have this feature with tsd so I can bundle d.ts files from repositories not included in DefinitelyTyped.

**Comment by wedneyyuri** [comment]
The ability to load from any git repo would be excellent. I believe that making this tool similar with the ```bower``` or ```npm``` (settings) has the effect of letting developers comfortable with this tool.

It also would help here: http://stackoverflow.com/questions/29857469/is-it-possible-to-reference-typescript-files-in-other-projects-without-adding-pr#comment47923706_29857469

**Comment by blakeembrey** [comment]
@wedneyyuri Yes, this is part of the proposal in #150. I agree that it's a pretty important feature moving forward.

**Comment by Zorgatone** [comment]
I would like this feature as well. I am using a .gitignore file that ignores the typings directory, and when I need definitions for external repos I have to place those in another directory :disappointed:

**Comment by blakeembrey** [comment]
@Zorgatone Thanks, yeah, it's a common issue and one I run into myself also. The next version currently implements a radically different dependency scheme (though, there needs more thought into any potential issues) and it allows you to do something like:

```json
"dependencies": {
  "my-dependency": "github:DefinitelyTyped/tsd/path/to/definition.d.ts"
}
```

**Comment by Zorgatone** [comment]
Nice. I think it's pretty sweet

**Comment by DovydasNavickas** [comment]
Any timeline and/or roadmap for a new version?

**Comment by wcandillon** [comment]
+1

**Comment by basarat** [comment]
FWIW the word on the street is *bower is dead* ... *long live npm*. 

Example : jquery installs
* on bower : http://bower.io/stats/ 198 
* on npm : https://www.npmjs.com/package/jquery 38,000

Of course take this with a pinch of salt as all statistics :rose:

**Comment by blakeembrey** [comment]
@basarat As right as you are about this, this issue is a comparison to Bower, not saying "from Bower" directly.

As for this feature now, there's not a TSD roadmap right now for providing this feature. I'm personally working on https://github.com/typings/typings which I intend to support installing ambient dependencies from DefinitelyTyped soon (https://github.com/typings/typings/issues/50 - right it already works manually (`github:DefinitelyTyped/DefinitelyTyped/node/node.d.ts#commit` for example), but this is an automated installation mechanism that'll resolve for you). The example I pasted above is the cruft of how `typings` works.
